README.md.gotmpl

```diff
@@ -320,9 +323,9 @@ to set for the specific environment variable. If a configuration option is requi
 environment declaration using `(required)` after its name and check if the variable is set in your installation script
 and break accordingly if not.
 
-If your feature needs a version specification, the recommended way is to use the environment variable
-`[FEATURE NAME]_VERSION`. This variable is also filled if the *CloudControl* user uses the FEATURES-variable approach
-to enable features.
+If your feature needs a version specification, set "requiresVersion" to true in the feature descriptor. This will
+enable the use of an environment variable `[FEATURE NAME]_VERSION`. This variable is also filled if the *CloudControl*
+user uses the FEATURES-variable approach to enable features.
 
```

Comment:
```suggestion
If you use the FEATURES variable, you can add the require version separated with a colon to desired feature.
```

Reply:
This section is about developing new features not the use of it. I'd rather keep the text as is.

Reply:
In this case I don't understand why the environment variable is described here. This also covers 'usage'.
But ok.